packages/preview/cumcm-muban/0.1.0/README.md

```diff
@@ -0,0 +1,68 @@
+cumcm-muban is a typst template for China Undergraduate Mathematical Contest in Modeling
+
+## Usage
+
+You can use this template in the Typst web app by clicking “Start from template” on the dashboard and searching for cumcm-muban.
+
+Alternatively, you can use the CLI to kick this project off using the command
+
+```
+typst init @preview/cumcm-muban
+```
+
+Typst will create a new directory with all the files needed to get you started.
+
+## Configuration
+
+This template exports the `cumcm` function with the following named arguments:
+
+- title: The thesis's title.
+- problem_chosen: The problem your team chosen.
```

Comment:
I recommend using kebab-case as it's the idiomatic Typst case convention.

Reply:
Thanks for your review, I have changed to use kebaby-case. Hope that's okay. 😄